tokio/src/stream/mod.rs

```diff
@@ -70,7 +70,7 @@ use take_while::TakeWhile;
 
 cfg_time! {
     mod timeout;
-    use timeout::Timeout;
+    pub use timeout::Timeout;
```

Comment:
**If** we expose these types, they should not be in the `tokio::stream` module as that would clog up API docs.

We could make a `tokio::stream::futures` module and export it there?